### ISSUE TITLE
eliminate redundant links in "events listing": 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#342]
At present, the "events listing" creates a link around the event title for each listed event. When there is a lead image associated with an event, it creates a separate link around the image. Like this:

```html
<div class="event-result-detail">
    <a href="http://isaw.nyu.edu/events/object-history">
        <img src="http://isaw.nyu.edu/events/object-history/leadImage_mini" alt="">
    </a>
    <a href="http://isaw.nyu.edu/events/object-history">
        <h3 class="event-title">Exhibition Gallery Talk: Object Histories</h3>
    </a>
     ᠁
</div>
```

Redundant links like this are a WCAG violation. As in other templates we've revised for accessibility, we need one and only one link to each event, so something like this would be ideal:

```html
<div class="event-result-detail">
    <a href="http://isaw.nyu.edu/events/object-history">
        <img src="http://isaw.nyu.edu/events/object-history/leadImage_mini" alt="">
        <h3 class="event-title">Exhibition Gallery Talk: Object Histories</h3>
    </a>
     ᠁
</div>
```

**Comment by cguardia** [comment]
Deployed to staging.

**Comment by cguardia** [comment]
Merged to master.
